.cbmc-batch/source/make_common_data_structures.c

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"). You may not use
+ * this file except in compliance with the License. A copy of the License is
+ * located at
+ *
+ *     http://aws.amazon.com/apache2.0/
+ *
+ * or in the "license" file accompanying this file. This file is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+ * implied. See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include <make_common_data_structures.h>
+
+void ensure_alg_properties_has_allocated_names(struct aws_cryptosdk_alg_properties *const alg_props) {
+    size_t md_name_size;
```

Comment:
see comment in https://github.com/aws/aws-encryption-sdk-c/pull/392

There seems to be some code duplication between these two reviews... in future revisions of these PRs, please structure them to avoid this duplication to ensure comments aren't lost.

Reply:
This PR is built on top of that one. Because of the build failures in CI we haven't been able to merge PRs, so they've built a string of dependencies.